### DISCUSSION TITLE
🐛 GsCore `BUG反馈汇总`

### DISCUSSION THREAD

**Discussion by wuchangjun233** [Genshin-bots/gsuid_core#3] [General]
我们了解您在使用过程中，遇到BUG的地方，欢迎在此留言！

BUG反馈格式
系统平台：
使用版本：
BUG描述：
问题截图：

**Comment by yhzcake** [comment]
系统平台：windows
使用版本：0.5.2
BUG描述：不能正确启动核心
问题截图：
![image](https://github.com/Genshin-bots/gsuid_core/assets/80627671/81669e75-6f28-4af9-a94d-2ad8fe95bc6f)

看起来是端口冲突,但实际上没有其他软件使用该端口
![image](https://github.com/Genshin-bots/gsuid_core/assets/80627671/e3a14f51-024b-4980-983a-e58465f826f8)

**Reply by KimigaiiWuyi** [reply]
看上去就是端口冲突
输入命令 `netstat -ano|findstr "8765"` 看看有没有进程占用8765端口

**Reply by yhzcake** [reply]
并没有,没有任何使用8765端口的软件
实际上第二张截图应该比较清晰,是按端口号排序的,可以看到8680直接跳到8790了
![image](https://github.com/Genshin-bots/gsuid_core/assets/80627671/131ddf96-5bca-4e51-9037-65bc8ea2f735)
